### PR TITLE
Update version 1.0.1-beta.1 for publish

### PR DESCRIPTION
**Pull request by shyunMin**
This PR is to update package version for resolving minor issue.

### CODE REVIEW

**Review by rookiejava — COMMENTED** [review]
Please check travis CI sever once again.

**Review by sung-su — APPROVED** [review]

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,6 @@
 language: node_js
 node_js:
   - "8"
-  - "10"
```
Comment (JoonghyunCho) [inline .travis.yml:4]: Is it ok to downgrade the version? which has been used as the build environment?

Comment (flyofsky) [reply .travis.yml:4]: just moved line.
Build tests of both nodes 8 and 10.

**Comment by flyofsky** [comment]
CI confirmed success.

**Review by flyofsky — APPROVED** [review]

**Pull request merged**